runtime/near-vm-runner-standalone/Cargo.toml

```diff
@@ -17,15 +17,17 @@ One can use `near-vm-runner-standalone` to test the smart contracts, e.g. with i
 to make sure it has expected behavior once deployed to the blockchain.
 """
 
-
 [dependencies]
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 clap = "2.33.0"
 base64 = "0.11"
+strum = "0.18"
+strum_macros = "0.18"
+num-rational = { version = "0.2.4" }
 
 near-vm-logic = { path = "../near-vm-logic", version = "1.0.0"}
-near-vm-runner = { path = "../near-vm-runner", version = "1.0.0" }
+near-vm-runner = { path = "../near-vm-runner", version = "1.0.0", features = ["wasmtime_vm"] }
```

Comment:
Typo?

Reply:
Nope, intentional, so that we could run in standalone mode with both VMs.